packages/react/src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -433,6 +433,7 @@ const TableToolbar = ({
           {hasSearch ? (
             <TableToolbarSearch
               {...search}
+              aria-label="search"
```

Comment:
Since this will be read by screen readers we should be using the translated string here.